client/sagas/night-mode-saga.js

```diff
@@ -29,6 +29,9 @@ export default function nightModeSaga(
     .flatMap(() => {
       const { app: { theme } } = getState();
       const newTheme = !theme || theme === 'default' ? 'night' : 'default';
+      if (typeof localStorage !== 'undefined') {
+        localStorage.setItem('fccTheme', newTheme);
+      }
```

Comment:
Instead of looking for the globalObject, use the store module.
See: https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/client/sagas/code-storage-saga.js#L2